packages/p2p/src/components/floating-rate/floating-rate.scss

```diff
@@ -214,9 +218,3 @@ button.dc-input-wrapper__button {
     display: flex;
     flex-direction: column;
 }
-
-.p2p-my-ads__form-field {
-    @include mobile {
-        height: auto !important;
-    }
-}
```

Comment:
is this okay to remove this style?

Reply:
I've checked the design, it is ok to remove the height since it is not effecting anywhere in the design